client/my-sites/checklist/wpcom-checklist/component.jsx

```diff
@@ -432,6 +434,41 @@ class WpcomChecklistComponent extends PureComponent {
 		);
 	};
 
+	renderDomainVerifiedTask = ( TaskComponent, baseProps, task ) => {
+		const { translate, siteSlug } = this.props;
+
+		return (
+			<TaskComponent
+				{ ...baseProps }
+				bannerImageSrc="/calypso/images/stats/tasks/create-tagline.svg"
+				buttonText={ translate( 'Verify' ) }
+				completedButtonText={ translate( 'Change' ) }
+				completedTitle={ translate( 'You verified the email address for your domain' ) }
+				description={ translate(
+					'The email address for your domain has to be verified to allow your domains to work properly.'
```

Comment:
We could use the following from the domains screen to prevent needing to translate new strings:

> We need to check your contact information to make sure you can be reached. Please verify your details using the email we sent you, or your domain will stop working.